tests/Api/Parser/JsonParserTest.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+use Aws\Api\Parser\Exception\ParserException;
+use Aws\Api\Service;
+use Aws\AwsClient;
+use GuzzleHttp\Promise\FulfilledPromise;
+use GuzzleHttp\Psr7\Response;
+use PHPUnit\Framework\TestCase;
+
+/*
+ * This class tests the PHP specific functionality of the JsonParser;
+ * cross-SDK testing is done in ComplianceTest.php
+ */
+
+/**
+ * @covers \Aws\Api\Parser\JsonRpcParser
+ * @covers \Aws\Api\Parser\JsonParser
+ */
+class JsonParserTest extends TestCase
+{
+    public function testUnixTimestampReturnedFromService()
```

Comment:
Should the name be more reflective of the actual test case, like `testUnixTimestampWhenModeledAsIso8601`?